.ci/pipeline/ci.yml

```diff
@@ -121,6 +127,9 @@ jobs:
       Python3.11_Sklearn1.2:
         PYTHON_VERSION: '3.11'
         SKLEARN_VERSION: '1.2'
+      Python3.11_SklearnMain:
+        PYTHON_VERSION: '3.11'
+        SKLEARN_VERSION: 'main'
```

Comment:
what if it will be updated with API break\?

Reply:
It's temporary scope for this PR (https://github.com/intel/scikit-learn-intelex/pull/1249/commits/0ae69474b8d92e3fe34333ca2c85f12b8f81ae44) and will be removed before merge